motion_odometry:
  ros__parameters:
    x_forward_scaling: 1.25
    x_backward_scaling: 0.95
    y_scaling: 0.9
    yaw_scaling: 1.0
    publish_walk_odom_tf: false
